openhexa/sdk/pipelines/parameter.py

```diff
@@ -343,3 +522,19 @@ def get_all_parameters(self):
             return [self.parameter, *self.function.get_all_parameters()]
 
         return [self.parameter]
+
+
+def is_connection_parameter(param: Parameter):
```

Comment:
This is fine - but we could do something cleaner, ideally encapsulating the logic that builds a connection instance in `Parameter.validate()`.

Do you think this would be possible?

Reply:
Yes, I've first tried that approach but for unknow (still searching) reason I couldn't load the connection config. But yes, I'll try to make it works.